safari/Shared.xcconfig

```diff
@@ -0,0 +1,2 @@
+MARKETING_VERSION = 6.8.0
```

Comment:
FYI Once this PR is merged, I'll enhance our `version.sh` script to increment this version as well.

Reply:
Perfect. More info about what’s missing here can be found on https://github.com/sindresorhus/refined-github/issues/3686 and more specifically https://github.com/sindresorhus/refined-github/issues/3766